### DISCUSSION TITLE
Turn off "debug mode"?

### DISCUSSION THREAD

**Discussion by RobbyRash** [tteck/Proxmox#3286] [Help Needed] [answered]
Getting this message in the system status page for Tandoor:
Debug Mode Warning
This application is still running in debug mode. This is most likely not needed. Turn of debug mode by setting DEBUG=0 int the .env configuration file.
How would I go about turning this off?

**Comment by tteck** [comment, marked as answer]
```
nano /opt/tandoor/.env
```

**Reply by RobbyRash** [reply]
Thanks!!
